policy/release_manager.md

```diff
@@ -0,0 +1,54 @@
+# Release manager role
+
+## Appointment
+
+A release manager (RM) is appointed for a specific upcoming major or minor release (i.e., x.y.z -> x.y+1.0 or x.y.z -> x+1.0.0).
```

Comment:
```suggestion
A release manager (RM) is appointed for a specific upcoming major or minor release (i.e., x.y.z -> x.y+1.0 or x.y.z -> x+1.0.0).

Because RM serves as an Assistant Release Manager (aRM) for prior maintenance release (see below), RM should be appointed 1 release cycle prior to the release he/she would be responsible for.
```

smth like that ;) i.e. if we consider @bpoldrack (as the one cutting 0.17.0 release) being responsible for `0.17` release cycle, we should have decided for RM for `0.18` right after `0.16.0` release since that is when RM-0.18 should serve as an aRM for 0.17 series whenever @bpoldrack is the RM.

Reply:
With that in mind, since both @bpoldrack did 0.17.0 and I did 0.17.1 both of us could be RMs for 0.17 series. I can do it, or would you @bpoldrack would like to ?  and then we need to decide on RM for 0.18 asap to follow this design